cypress/support/commands.ts

```diff
@@ -14,23 +14,44 @@ import { Result } from "axe-core";
 import "@cypress/code-coverage/support";
 import "cypress-axe";
 
+const loginWithRetry = (iteration: number = 0): void => {
+    if (iteration >= 5) {
+        // This only ever seems to happen on GH Actions on the first test, so we can just log and move on
+        // Redirects are tested by the auth spec
+        cy.log("Login redirect failed");
+        return;
+    }
+
+    cy.get("h1").then(($h1) => {
```

Comment:
Can you explain what the `$` is for?

Reply:
This is a jquery idiom (cypress uses it under the hood). I'm not sure if we want to keep them or not, but this is generally what jquery code looks like.

https://jquery.com/
(jquery was a very commonly used library that is now largely redundant due to the improvement of browsers)

Reply:
I think I prefer the variable naming to be clearer and without the `$` sign.